tests/api_connexion/endpoints/test_event_log_endpoint.py

```diff
@@ -29,17 +34,177 @@ def setUpClass(cls) -> None:
 
     def setUp(self) -> None:
         self.client = self.app.test_client()  # type:ignore
+        with create_session() as session:
+            session.query(Log).delete()
+        self.default_time = "2020-06-10T20:36:40.186116+00:00"
```

Comment:
Can you reduce the precision on the dates given? Now it is difficult to compare them.